src/v2i-hub/CARMAStreetsPlugin/src/J2735ToSRMJsonConverter.h

```diff
@@ -0,0 +1,16 @@
+#include "jsoncpp/json/json.h"
+#include <tmx/j2735_messages/SignalRequestMessage.hpp>
+
+namespace CARMAStreetsPlugin
+{
+    class J2735ToSRMJsonConverter
+    {
+    private:
+        const std::string MsgType = "SRM";
+
+    public:
+        J2735ToSRMJsonConverter();
```

Comment:
Can we add some comments

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/518/commits/b63bfbec39f57ad20a45bff5d8c0595b69cc64e9